docs/node-operators/get-started/setup-datanode.md

```diff
@@ -307,7 +322,7 @@ vega datanode postgres run --home="YOUR_DATA_NODE_HOME_PATH"
 In either case, the files for the database will be stored in the data node `state` folder located at `YOUR_DATA_NODE_HOME_PATH/state.data-node/storage/sqlstore`.
 
 ### Buffered event source
```

Comment:
Does this need to be in the doc I wonder?  there's a lot of info in the doc, the defaults for this are generally fine, anything we can do to keep the doc simple as possible is probably good as long as we think its not important for the operator to know about it, this is almost implementation detail, so I think falls into that category, thoughts?

Reply:
I also think it is helpful for the end user in case they need to troubleshoot.

Reply:
it creates the error in the state directory, which must have write perms for other things, and will create the dir at startup if not exists, so should be fine